src/containers/WaveformContainer.js

```diff
@@ -130,6 +131,14 @@ class WaveformContainer extends Component {
   }
 }
 
+WaveformContainer.propTypes = {
+  structureURL: PropTypes.string.isRequired,
+  waveformURL: PropTypes.string.isRequired,
+  audioURL: PropTypes.string.isRequired,
+  streamDuration: PropTypes.number.isRequired,
+  initStructure: PropTypes.object,
```

Comment:
Should this be required as well?
```suggestion
  initStructure: PropTypes.object.isRequired,
```